course/meetings/mie-matthew-mackay-2017-03-13.md

```diff
@@ -0,0 +1,50 @@
+
+---
+contact: Matthew Mackay
+department: Mechanical and Industrial Engineering
+uoftcoders: Madeleine, Joel
+date: 2017-03-13
+---
+
+# Summary
+- His ideal course would be somewhat engineering specific and attractive to both mechanical and industrial engineers. He seemed more interesting in uniting these two engineering streams than bridging gaps to other departments.
+    - If there was a course that bridged the interests of the Mechanical and Industrial streams, that would be very attractive to the department.
+- Want more programming in general for his undergrads.
+    - Currently a big gap between taking a few courses in first year and then basically nothing until fourth year.
+        - At least for Mechanical engineers where programming is a hard sell, the industrial stream has decent programming experience.
+    - Currently mostly dependent on student electives.
+- The RQM could potentially replace and update their numerical methods course.
```

Comment:
Seems to be 3rd year http://coursefinder.utoronto.ca/course-search/search/courseInquiry?methodToCall=start&viewId=CourseDetails-InquiryView&courseId=MIE334H1S20171